workshop/task_01/task_01.md

```diff
@@ -81,11 +81,14 @@ scrapy crawl kinoprogramm -o ../data/kinoprogramm.json
 
 Data will be written to the file specified with the `-o` parameter. In our case: `<project-dir>/scrapy/data/kinoprogramm.json`.
 
+When this command is finished, the scraping and parsing of the data is done and saved in the newly created file `scrapy/data/kinoprogramm.json`. Check that it is there and includes cinema data.
```

Comment:
I would remove line 84, since the "instruction" to look in the data is already below (line 89).
```suggestion
```